### PR TITLE
chore: show unsupported error screen for gated content

### PR DESCRIPTION
**Pull request by saeedbashir**
This PR address the following Issue

```
"Learn > Courses > Any audit track course (ex: IBM's The Data Science Method) > Course Home

Tabbing on locked content (such as graded assignments and quiz) in the module doesn't prompt anything"

### CODE REVIEW

**Comment by rnr** [comment]
@saeedbashir please look on tests

File: Course/Course/Presentation/Unit/Subviews/UnknownView.swift [hunk 1]
```diff
@@ -9,33 +9,37 @@ import SwiftUI
 import Core
 import Theme
 
-struct UnknownView: View {
+public struct UnknownView: View {
```
Comment (rnr) [inline Course/Course/Presentation/Unit/Subviews/UnknownView.swift]: maybe better name is 'NotAvailableOnMobileView' or something like that?

File: Course/Course/Presentation/Unit/Subviews/UnknownView.swift [hunk 2]
```diff
@@ -9,33 +9,37 @@ import SwiftUI
...
+                    .font(Theme.Fonts.titleLarge)
+                    .multilineTextAlignment(.center)
+                    .padding(.top, 40)
+                Text(CourseLocalization.NotAvaliable.description)
+                    .font(Theme.Fonts.bodyLarge)
+                    .multilineTextAlignment(.center)
+                    .padding(.top, 12)
+                StyledButton(CourseLocalization.NotAvaliable.button, action: {
+                    if let url = URL(string: url) {
+                        UIApplication.shared.open(url)
```
Comment (rnr) [inline Course/Course/Presentation/Unit/Subviews/UnknownView.swift]: maybe check if canOpen url also?

**Review by rnr — APPROVED** [review]

File: Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift [hunk 3]
```diff
@@ -95,7 +95,10 @@ struct CustomDisclosureGroup: View {
                                                 guard let chapterIndex = chapterIndex else { return }
                                                 guard let sequentialIndex else { return }
                                                 guard let courseVertical = sequential.childs.first else { return }
```
Comment (shafqat-muneer) [inline Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift:97]: Should we also display any toast error messages to the user in these possible cases?

**Review by shafqat-muneer — CHANGES_REQUESTED** [review]

File: Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift (hunk 3 above)
Comment (saeedbashir) [reply Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift:97]: We will he handing gated content properly post MVP, so we will look into it as well.

**Review by shafqat-muneer — APPROVED** [review]
LGTM 👍

**Pull request merged**